tools/github_readme_sync/readme.py

```diff
@@ -258,16 +274,24 @@ def replace_image(match):
             if any(ignore_image in match.groups()[1] for ignore_image in IGNORE_IMAGES):
                 return match.group(0)
             alt_text, image_src = match.groups()
+
+            # Split image source and fragment
+            src_parts = image_src.split("#")
+            clean_src = src_parts[0]
+            style = "border-radius: 8px;"
+            if len(src_parts) > 1:
+                style = f"{style}{src_parts[1]}"
```

Comment:
**issue** I think we should specify key value pairs in Markdown, and not, as this suggest, inline CSS.